static_src/stores/service_instance_store.js

```diff
@@ -150,18 +161,27 @@ class ServiceInstanceStore extends BaseStore {
           action.spaceGuid,
           action.servicePlanGuid
         );
+        this._createLoading = true;
+        this.emitChange();
         break;
       }
 
       case serviceActionTypes.SERVICE_INSTANCE_CREATED: {
         cfApi.fetchServiceInstance(action.serviceInstance.guid);
-        this._createInstanceForm = null;
+        this._createLoading = false;
+        this._createdTempNotification = true;
         this.emitChange();
+        setTimeout(() => {
+          this._createInstanceForm = null;
+          this._createdTempNotification = false;
+          this.emitChange();
+        }, CREATED_NOTIFICATION_TIME_MS);
```

Comment:
I think we're in agreement that we should move all the cfApi calls to the actions.